### PR TITLE
feat(sandbox): Initial Layout

### PR DESCRIPTION
**Pull request by jennmedellin**
## Description
Sandbox demoing best practices for layouts and patterns
Note: Moving resources to new templates folder
Missing: layouts and patterns

### What's included?
- Overview for layouts & patterns

#### Test Steps
- [ ] `yarn install`
- [ ] `ng serve`
- [ ] Test overview & verify code

#### Definition of Done
- [ ] Adheres to [Covalent](https://teradata.github.io/covalent/), [Angular-Material](https://material.angular.io/) & [Material Design](https://material.io/guidelines/) patterns
- [ ] Keyboard `a11y` support (are all elements accessible through key presses)
- [ ] Resize browser to test responsive breakpoints
- [ ] `yarn lint` passes

screen shot 2019-01-07 at 6 43 49 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+  "name": "covalent-resources",
```
Comment (ghost) [inline package.json]: Do we want to change this to `covalent-sandbox`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  "version": "1.0.0",
+  "private": true,
+  "description": "Resources for docs, demos and sketch",
```
Comment (ghost) [inline package.json]: Not sure if we wanted to change this too? the description line.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  },
+  "engines": {
+    "node": ">=8.9.4",
+    "npm": ">=5",
+    "yarn": ">=1.3.2"
+  },
+  "repository": "https://github.com/teradata/covalent-resources.git",
+  "license": "MIT",
+  "author": "Teradata UX",
+  "contributors": [
```
Comment (ghost) [inline package.json:33]: Do we want to update the contributors list too?

File: scripts/ghpages-deploy [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+cp -r dist/* .
+echo 'Copied dist/ directory into root dir'
+
+echo 'Tracking files'
+git add -A .
+echo 'Commiting files'
+git commit -a -m 'gh-pages update'
+echo 'Pushing files into gh-pages branch'
+git push origin gh-pages --force
```
Comment (ghost) [inline scripts/ghpages-deploy:25]: Were we planning to serve covalent sandbox though github pages?

File: .gitignore [hunk 5]
```diff
@@ -1 +1,2 @@
 .DS_Store
+node_modules/
```
Comment (ghost) [inline .gitignore:2]: General question - did you pull the [Covalent Quickstart](https://github.com/Teradata/covalent-quickstart) files into here or was it a different project?

Comment (jennmedellin) [reply .gitignore]: Yes, these are from QuickStart

Comment (jennmedellin) [reply .gitignore:2]: Yes, these are from QuickStart

File: scripts/ghpages-deploy (hunk 4 above)
Comment (jennmedellin) [reply scripts/ghpages-deploy:25]: Yes, once we get through a few patterns we’ll host in gh-pages

File: package.json [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    "@angular/compiler": "7.0.0",
+    "@angular/core": "7.0.0",
+    "@angular/forms": "7.0.0",
+    "@angular/http": "7.0.0",
+    "@angular/material": "7.0.1",
+    "@angular/platform-browser": "7.0.0",
+    "@angular/platform-browser-dynamic": "7.0.0",
+    "@angular/platform-server": "7.0.0",
+    "@angular/router": "7.0.0",
+    "@covalent/core": "2.0.0-beta.5",
```
[the remark this answers is not in the document]
Comment (jennmedellin) [reply package.json]: Sure! If you have time, else I’ll give it a shot later tonight

[the remark this answers is not in the document]
Comment (jennmedellin) [reply package.json]: Sure! If you have time, else I’ll give it a shot later tonight

File: .gitignore (hunk 5 above)
Comment (ghost) [reply .gitignore:2]: Ah okay, it was recently updated - this repo can be updated in a future PR to keep it the scope of it smaller.

**Review by ghost — APPROVED** [review]
LGTM (besides upgrading to latest Angular/Covalent)

**Comment by JulieMarie** [comment]
LGTM

**Pull request merged**